src/app/pages/buy/components/onramp-provider-layout.tsx

```diff
@@ -3,17 +3,20 @@ import { Button, Stack } from '@stacks/ui';
 import { openInNewTab } from '@app/common/utils/open-in-new-tab';
 import { Caption, Title } from '@app/components/typography';
 import { SpaceBetween } from '@app/components/space-between';
+import { BuyTokensSelectors } from '@tests/page-objects/buy-tokens-selectors';
 
 const providersInfo = {
   transak: {
     title: 'Transak',
     body: 'Non-US residents can purchase STX with credit card, debit card, or bank transfer via Transak.',
     cta: 'Buy on Transak',
+    test_id: 'BtnTransak',
```

Comment:
Naming convention is to use camelCase here
```suggestion
    testId: 'BtnTransak',
```